weaviate/templates/querierDeployment.yaml

```diff
@@ -0,0 +1,456 @@
+{{ if .Values.experimental.querier.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: querier
```

Comment:
I would prefix all of our services with `weaviate` so the name that I suggest here would be then `weaviate-querier` and all of the labels also should have that prefix to easily find "our" services / deployments